.github/workflows/issue-management-stale-action.yml

```diff
@@ -5,11 +5,17 @@ on:
     # hourly at minute 23
     - cron: "23 * * * *"
 
+permissions:  # added using https://github.com/step-security/secure-repo
+  contents: read
```

Comment:
should we add this to top-level of all workflows?

Reply:
That’s generally the OSSF recommandation indeed